client/data/types.d.ts

```diff
@@ -11,4 +11,5 @@ export interface State {
 	capital?: CapitalState;
 	paymentIntents?: PaymentIntentsState;
 	files?: FilesState;
+	paymentActivity?: any;
```

Comment:
I would encourage us to [avoid `any`](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/general-conventions.md#avoid-any) here and type this as accurately as possible since we know/control the shape of this state.

```suggestion
	paymentActivity?: PaymentActivityState;
```

Reply:
Good catch, made changes here https://github.com/Automattic/woocommerce-payments/pull/8589/commits/2a0d4ebea23fba0d19690da3d86b0c3919a7317e